pkg/skel/skel.go

```diff
@@ -241,6 +277,17 @@ func (t *dispatcher) pluginMain(cmdAdd, cmdCheck, cmdDel func(_ *CmdArgs) error,
 		return types.NewError(types.ErrIncompatibleCNIVersion, "plugin version does not allow CHECK", "")
 	case "DEL":
 		err = t.checkVersionAndCall(cmdArgs, versionInfo, cmdDel)
+		if err != nil {
+			return err
+		}
+		if cmdArgs.NetnsOverride != "TRUE" {
```

Comment:
@Poor12 could we make this a case-insensitive comparison for "true", and also add a check for "1"?

Reply:
make it.